### PR TITLE
Prep for 2.0

### PR DESCRIPTION
**Pull request by todb**
This finishes off #207 by creating a new gemspec for publishing.

Need to write some release notes (it's really just the one line summarizing #207, nothing else has changed), then this'll be good to land if someone else eyeballs it (@adfoster-r7 or @claudijd, I'll take either!).

### CODE REVIEW

**Comment by todb** [comment]
Well that's a great big pile of fail.

**Comment by todb** [comment]
All right, this looks good. I haven't enabled branch protection yet, but sure could use an eyeball review from @claudijd or @adfoster-r7 to just see that I'm not doing anything real dumb in here.

**Comment by todb** [comment]
Incidentally, I've remembered how to publish to RubtyGems. I of course had to re-up my signing cert since the machine I had it on several years ago is long gone, but that's committed here: 28bb3534858a87255cc8eeee428692024b47174a

You can tell it's me because of the signed commit.

**Comment by todb** [comment]
@adfoster-r7 I'll trade you a gem push for a code review, in other words.

File: .github/workflows/verify.yml [hunk 1]
```diff
@@ -51,6 +51,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
```
Comment (adfoster-r7) [inline .github/workflows/verify.yml:54]: Is there any more context on why this change was needed? 👀 

From a quick Google it looks like folk are adding this when they're wanting their github action to work with `pull_request_target` - which isn't the case here:

https://github.com/packetfu/packetfu/blob/bc7362f7c3d8e40e2c8bdeaab76b7c17506306c1/.github/workflows/verify.yml#L19-L25

**Comment by adfoster-r7** [comment]
All looks good to me; Just a question on the Github action code change :+1:

**Comment by todb** [comment]
Cool cool. Yeah the action change was required to notice that the PR was coming from todb/packetfu and not packetfu/packetfu. If you look at the first chunk of failures, you'll see the "can't find a branch or tag ref" messages. That's why.

File: .github/workflows/verify.yml [hunk 2]
```diff
@@ -51,6 +51,7 @@ jobs:
...
           ref: ${{ github.event.pull_request.head.ref }}
```
Comment (adfoster-r7) [inline .github/workflows/verify.yml:55]: I believe this should fix the original CI issues you hit and ensure that the merge commit is being tested against the current repo instead of the user's fork:

```suggestion
```

Comment (todb) [reply .github/workflows/verify.yml:55]: Let's find out!

**Comment by adfoster-r7** [comment]
Thanks! Noticed a few other things snuck in the version bump from before my time and maybe a few tweaks to the gemspec ignore config could be added - but all looks good 💯

https://my.diffend.io/gems/packetfu/1.1.13/2.0.0

**Comment by todb** [comment]
Yep, a few years of changes. We've not updated in... a while :)

https://github.com/packetfu/packetfu/releases/tag/version-2.0.0

**Comment by todb** [comment]
I wonder how @adfoster-r7 managed to dodge a Contributors badge, btw. A master of stealth. 🥷
